DataFormats/Portable/interface/PortableDeviceCollection.h

```diff
@@ -0,0 +1,76 @@
+#ifndef DataFormats_Portable_interfacePortableDeviceCollection_h
+#define DataFormats_Portable_interfacePortableDeviceCollection_h
```

Comment:
Should these be
```suggestion
#ifndef DataFormats_Portable_interface_PortableDeviceCollection_h
#define DataFormats_Portable_interface_PortableDeviceCollection_h
```
?

Reply:
Yes.